### ISSUE TITLE
U+1BAB SUNDANESE SIGN VIRAMA does not form conjuncts

### ISSUE THREAD

**Issue by dscorbett** [notofonts/sundanese#2]
### Font
NotoSansSundanese-Regular.ttf

### Where the font came from, and when
Site: https://noto-website-2.storage.googleapis.com/pkgs/NotoSansSundanese-unhinted.zip
Date: 2018-03-19

### Font version
Version 1.05 uh

### Issue
U+1BAB SUNDANESE SIGN VIRAMA should form conjuncts. Currently, it appears as a small plus sign overstriking the following glyph. See [L2/09-251R](http://www.unicode.org/L2/L2009/09251r-n3666r-sundanese.pdf) for examples of conjuncts.

### Character data
ᮊ᮫ᮒ
U+1B8A SUNDANESE LETTER KA
U+1BAB SUNDANESE SIGN VIRAMA
U+1B92 SUNDANESE LETTER TA

### Screenshot
![x](https://user-images.githubusercontent.com/1124347/37609901-ab2eec64-2b74-11e8-9ebb-b3946ba2d6b8.png)

**Comment by Ilhamkang** [comment]
I agree to add conjuncts based L2/09-251R document. I will be glad to check all sequences.

**Comment by PaulHanslow** [comment]
hey @Ilhamkang, by conjuncts I'm assuming you mean the following 24? If not please provide information on what's missing from Google Noto Sudanese. 

![Screenshot 2024-08-18 at 5 03 29 PM](https://github.com/user-attachments/assets/b3ced640-931d-4280-8ba2-cf735477bc77)

**Comment by ghost** [comment]
Hi, @PaulHanslow @simoncozens . I strongly object to this approach, particularly regarding the glyphs being proposed. The [L2/09-251R](https://www.unicode.org/L2/L2009/09251r-n3666r-sundanese.pdf) document cannot be fully relied upon, as many of the ligatures included are not historically attested. Additionally, due to the limited references available at the time of its compilation (2009), the document lacks credibility, especially since it only accounts for 24 ligatures.
Based on historical texts and [recent researchs](https://tufs.repo.nii.ac.jp/record/5172/files/lacs026008.pdf), conjunct forms (_pasangan_) in the Sundanese script are essentially identical to their base letters, except for U+1B9D, U+1B9C, U+1B9B, U+1B9A, and U+1B99, which already have encoded _pasangan_ forms. Furthermore, the correct position of these conjunct forms is directly below the main letter, not at the bottom-right as stated in L2/09-251R.
The concept of conjunct forms (_pasangan_) in the Sundanese script follows the same principles as in Kawi, Javanese, and Balinese scripts, where _pasangan_ can carry signs both above and below, and they do not attach to the letter above them. The following characters have conjunct forms: U+1B83 - U+1BA0 and U+1BBB - U+1BBC.

Here the examples of the application:
<img width="534" alt="image" src="https://github.com/user-attachments/assets/b8dbb292-60c9-4772-b27c-0c6648d01eed" />

The use of below signs
![image](https://github.com/user-attachments/assets/fdc20744-159e-483f-b877-5ee651010cbc)

Thankyou.
P.S.: please also fix [it](https://github.com/notofonts/sundanese/issues/9) too.

**Comment by PaulHanslow** [comment]
Hi @niomi13, thanks for your comments.

I could find little search on the 'correct' conjunct design after extensive research. I'd be happy to reassess the shapes if/when Google and myself have time to do so. What you have provided is very very helpful to improving the font and I greatly appreciate the time you've taken in writing.

**Comment by ghost** [comment]
Hi @PaulHanslow, thanks for your response. I truly appreciate your willingness to reassess the conjunct designs when time allows.  
As designers, you undoubtedly have a deeper understanding of the appropriate proportions for conjunct forms. As any ordinary users, we primarily expect the conjuncts to have the same shape as their base letters.
Once again, I appreciate your time and effort in improving the font. Looking forward to future developments!

Here you can see how good Sundanese font applied (especially conjunct, final m, and others Old Sundanese characters): https://www.behance.net/gallery/80070849/Sundanese-font-Kataruman
